src/app/components/NewDocumentContestButton/style.scss

```diff
@@ -0,0 +1,31 @@
+:local{
+    .stylebtn{
+        display: grid;
+        grid-template-columns: 100%;
+        grid-template-rows: 50% 50%;
+        color: #1c83c6;
+        width: 112px !important;
+        height: 80px;
+        border: 2px solid #1c83c6;
+        border-right: 2px solid rgba(28, 131, 198, 0.4);
+        border-left: 0;
+        position: relative;
+        flex-wrap: nowrap;
+        div{
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+        div:nth-child(1){
+            align-items: flex-end;
+        }
+        div:nth-child(2) 
+        {
+            align-items: flex-start;
+            p{
+                font-size: 16px;
+                cursor: pointer;
```

Comment:
Nên set cursor cho cả nút thay vì chỉ có text nha em, tức là khi em đưa chuột vào trong area của cái nút thì nó phải hiện con trỏ, chứ không phải chỉ mỗi text. Đồng nghĩa với việc khi ấn nút thì cũng phải cho phép người dùng ấn toàn bộ nút, thay vì hiện tại chỉ ấn được mỗi text